lib/acts_as_follower.rb

```diff
@@ -6,25 +6,5 @@ module ActsAsFollower
   autoload :FollowerLib,  'acts_as_follower/follower_lib'
   autoload :FollowScopes, 'acts_as_follower/follow_scopes'
 
-  def self.setup
```

Comment:
This change is not backward compatible. This method should be retained and a deprecation should be emitted